src/tests/multihost/alltests/test_backtrace.py

```diff
@@ -76,6 +78,13 @@ def test_0001_bz2021196(self, multihost, backupsssdconf):
         tools.sssd_conf(section, domain_params, action='delete')
         tools.clear_sssd_cache()
         logfile = f'/var/log/sssd/sssd_{ds_instance_name}.log'
+        log_str = multihost.client[0].get_file_contents(logfile). \
+            decode('utf-8')
+        print(f'\n{logfile}\n+===++++++===+\n{log_str}\n')
```

Comment:
why do we need to print the log here?

Reply:
It helps by saving time for checking what changed in the logs in case test fails.